src/runtime/ProxyObject.h

```diff
@@ -78,7 +78,8 @@ class ProxyObject : public Object {
     // http://www.ecma-international.org/ecma-262/5.1/#sec-8.6.2
     virtual const char* internalClassProperty() override
     {
-        return "Proxy";
+        ASSERT(m_target);
```

Comment:
This assertion is invalid, see https://github.com/Samsung/escargot/pull/477#discussion_r341532308.